app/styles/notification.css

```diff
@@ -84,6 +84,7 @@
 .text-title {
   font-weight: bold;
   font-size: 20px;
+  padding-bottom: 4px;
```

Comment:
Instead of using `padding-bottom` here, use `gap` on parent container.
That's a good practice.

Reply:
The parent is not a flex. And when it is changed to flex, then the CSS is breaking. 

![image](https://user-images.githubusercontent.com/41564532/189718452-3e90ed51-effc-405a-874e-70aff9c098ed.png)

Reply:
> The parent is not a flex. And when it is changed to flex, then the CSS is breaking.
> 
> ![image](https://user-images.githubusercontent.com/41564532/189718452-3e90ed51-effc-405a-874e-70aff9c098ed.png)

@rajat-mehra05

Reply:
Okay got it.